ui/src/components/erc/erc.css

```diff
@@ -1,3 +1,11 @@
 .Erc {
   text-align: center;
 }
+
+#PreviewNotice {
+  color: white;
+  background-color: #CE5100;
```

Comment:
You added the color here again as css. It is also possible to use the color, specified in the theme.js. This would be the better way. Because If we want to change the color, we just have to change the color at one place and not in every css file. I just have inspected the UI and saw that there is also some code implemented this way. I have created a new issues #197, maybe we can postpone this and check, update the whole UI afterwards.